docs/src/main/paradox/docs/getting-started/running-nexus/docker-swarm/docker-compose.yaml

```diff
@@ -1,45 +1,79 @@
 version: "3.3"
 services:
   delta:
-    image: bluebrain/nexus-delta:latest
-    entrypoint: ["bin/wait-for-it.sh", "-s", "-t", "0", "cassandra:9042", "--", "./bin/delta",
-                 "-Xms1G",
-                 "-Xmx2G",
-                 "-Dapp.instance.interface=0.0.0.0",
-                 "-Dapp.http.interface=0.0.0.0",
-                 "-Dapp.http.public-uri=http://localhost:8080",
-                 "-Dapp.elastic-search.base=http://elasticsearch:9200",
-                 "-Dapp.sparql.base=http://localhost:9999/blazegraph",
-                 "-Dakka.persistence.cassandra.journal.keyspace-autocreate=true",
-                 "-Dakka.persistence.cassandra.journal.tables-autocreate=true",
-                 "-Dakka.persistence.cassandra.snapshot.keyspace-autocreate=true",
-                 "-Dakka.persistence.cassandra.snapshot.tables-autocreate=true",
-                 "-Ddatastax-java-driver.basic.contact-points.1=cassandra:9042"]
+    depends_on:
+      - elasticsearch
+      - blazegraph
+      - cassandra
+    image: bluebrain/nexus-delta:1.5.0
+    environment:
+      DELTA_PLUGINS: "/opt/docker/plugins/"
+      KAMON_ENABLED: "false"
+    entrypoint: [ "bin/wait-for-it.sh", "-s", "-t", "0", "cassandra:9042", "--", "./bin/delta-app",
+                  "-Xmx2G",
+                  "-Dapp.http.interface=0.0.0.0",
+                  "-Dapp.http.base-uri=http://delta:8080/v1",
+                  "-Dapp.cluster.remote-interface=127.0.0.1",
+                  "-Dapp.cluster.seeds=127.0.0.1:25520",
+                  "-Dapp.database.cassandra.contact-points.1=cassandra:9042",
+                  "-Dapp.database.cassandra.keyspace-autocreate=true",
+                  "-Dapp.database.cassandra.tables-autocreate=true",
+                  "-Dplugins.elasticsearch.base=http://elasticsearch:9200",
+                  "-Dplugins.blazegraph.base=http://blazegraph:9999/blazegraph",
+                  "-Dakka.persistence.cassandra.events-by-tag.first-time-bucket=20210503T00:00",
+                  "-Dakka.persistence.cassandra.events-by-tag.eventual-consistency-delay=4s",
+                  "-Dakka.persistence.cassandra.query.refresh-interval=1s"
+    ]
+
   elasticsearch:
-    image: "docker.elastic.co/elasticsearch/elasticsearch:7.4.0"
+    image: "docker.elastic.co/elasticsearch/elasticsearch:7.12.0"
     environment:
-      ES_JAVA_OPTS: "-Xms2G -Xmx2G"
+      ES_JAVA_OPTS: "-Xmx1G"
       discovery.type: "single-node"
       bootstrap.memory_lock: "true"
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:9200/_cluster/health" ]
+      interval: 30s
+      timeout: 20s
+      retries: 3
+    ports:
+      - "9200"
 
   cassandra:
```

Comment:
It can be done later but for we should add examples with PostgreSQL. And it should even replace the ones with Cassandra for running Delta on a single node.